package.json

```diff
@@ -41,6 +41,7 @@
   },
   "scripts": {
     "test": "mocha",
+    "debugtest": "mocha --debug-brk $npm_package_options_mocha",
```

Comment:
Thanks for taking the time to review the PR. Could you elaborate on what is wrong with the task `debugtest`? Not sure whats wrong there...